nautilus_trader/portfolio/portfolio.pyx

```diff
@@ -1247,24 +1269,24 @@ cdef class Portfolio(PortfolioFacade):
         return Money(total_pnl, currency)
 
     cdef Price _get_last_price(self, Position position):
-        cdef PriceType price_type
+        price: Price | None = None
```

Comment:
As above, lets just keep the changes for this PR to minimally what is needed for bars to be able to update.